BlogPosts/BEMnaming.html

```diff
@@ -0,0 +1,91 @@
+<!--FOR ALL NONHOME PAGES-->
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <title></title>
+  <link rel="stylesheet" href="../styles.scss" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+
+<body>
+  <header class="nav__img-small">
+    <nav class="navbar">
+      <div class="nav__navbar">
+        <Button class="nav__button">
+          <a href="../index.html" class="link--primary">
+            <b>Home</b>
+          </a>
+        </Button>
+        <Button class="nav__button">
+          <a href="../Blogs.html" class="link--primary">
+            <b>Blogs</b>
+          </a>
+        </Button>
+        <Button class="nav__button">
+          <a href="../About.html" class="link--primary">
+            <b>About</b>
+          </a>
+        </Button>
+      </div>
+    </nav>
+  </header>
+</body>
+<!--FOR ALL NONHOME PAGES-->
+
+<body style="background-color: #ECECEA;">
```

Comment:
We never want to do handle styling in the html - you can either make a class for this or just stick that color right onto the body element.